sat/meshed.go

```diff
@@ -26,6 +29,13 @@ func (s *Sat) handleFnResult(msg grav.Message, result interface{}, fnErr error)
 	ctx.UseRequestID(req.ID)
 	ctx.UseScope(loggerScope{req.ID})
 
+	spanCtx, span := s.tracer.Start(ctx.Context, "handleFnResult", trace.WithAttributes(
```

Comment:
If we wrap listenAndRun, then what are we tracing? That way every single request (and I mean all of them) will end up becoming a child trace to the one parent trace, which isn't going to be useful here.

My assumption is that a trace is for a single request, a single job, a single something rather than the entire lifetime of the service.

Reply:
I would see it as each time a message is recieved, it goes through a "middleware" before being handled by Reactr.

It'll be a larger change possibly involving changes in other repos which is why I think it's not something to do in this PR.